doc/changelog/04-tactics/19060-replace-with-diropt.rst

```diff
@@ -0,0 +1,6 @@
+- **Added:**
+  the :tacn:`replace` tactic now accepts an orientation
+  when used with a `with` clause
```

Comment:
```suggestion
  the first form of the :tacn:`replace` tactic (with two terms) now accepts `->` and `<-`
  to specify the direction of the replacement
```

or arrows

Reply:
Honestly this is less informative as one has no context around to know what the first form of replace could be.

Reply:
```suggestion
  the :tacn:`replace` tactic now accepts `->` and `<-`
  to specify the direction of the replacement
  when used with a `with` clause
```
"orientation" is not really the right word here

The meaning of "first form" is clear if you click to see the documentation BTW, we use that wording in the doc for replace and elsewhere.

Reply:
Done.